predicate-builder-service/src/index.ts

```diff
@@ -18,8 +17,8 @@ app.post('/debug', (req, res) =>
 );
 
 
-app.listen(config.hostPort, () => {
+app.listen(3000, () => {
   console.log(
-    `Predicate Builder Service - Listening on: http://localhost:${config.hostPort}`,
+    `Predicate Builder Service - Listening on: http://localhost:${3000}`,
```

Comment:
Why is this being hardcoded? does it has to do with openfaas needing an specific port?

Reply:
This is hardcoded because openfaas has a connection to your server (on 3000) internally using their watchdog